src/ts/App/App.spec.tsx

```diff
@@ -1,16 +1,18 @@
-// import 'mocha'
-// import * as React from 'react'
-// import * as ReactDOM from 'react-dom';
-// import { expect } from 'chai'
-// import { mount, shallow } from 'enzyme'
-// import App from './index'
-// import * as sinon from 'sinon'
+import 'mocha';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import * as chai from 'chai';
+import { mount, shallow } from 'enzyme';
+import App from './index';
+import AppRouter from '../Router';
+import * as sinon from 'sinon';
+import * as sinonChai from 'sinon-chai';
+const expect = chai.expect;
+chai.use(sinonChai);
 
-// describe('<App />', () => {
-//   it('calls componentDidMount', () => {
-//     sinon.spy(App.prototype, 'componentDidMount');
-//     const wrapper = mount(<App />)
-//     console.log(wrapper.html())
-//     expect(App.prototype.componentDidMount.calledOnce).to.equal(true)
-//   })
-// })
+describe('<App />', () => {
+  it('should render AppRouter', () => {
+    const wrapper = mount(<App />)
+    expect(wrapper.find(AppRouter)).to.have.length(1);
```

Comment:
make sure that the negative version of this test fails, i.e.:
```
expect(wrapper.find(AppRouter)).not.to.have.length(0);
```
or a working equivalent :D